wkconnect/backends/boss/client.py

```diff
@@ -0,0 +1,143 @@
+        # {
+        #     "name": "ara_2016_50um",
+        #     "description": "50 um Allen reference atlas",
+        #     "x_start": 0,
+        #     "x_stop": 264,
+        #     "y_start": 0,
+        #     "y_stop": 160,
+        #     "z_start": 0,
+        #     "z_stop": 228,
+        #     "x_voxel_size": 50.0,
+        #     "y_voxel_size": 50.0,
+        #     "z_voxel_size": 50.0,
+        #     "voxel_unit": "micrometers"
+        # }
+        coord_url = f"{domain}/v1/coord/{coord_frame}"
+        async with await self.auth_get(coord_url, token_key) as r:
+            return await r.json()
+
+    async def get_channel(
+        self,
+        domain: str,
+        collection: str,
+        experiment: str,
+        channel: str,
+        token_key: Key,
+    ) -> JSON:
+        # {
+        #     "name": "nissl_50um",
+        #     "description": "",
+        #     "experiment": "sagittal_50um",
+        #     "default_time_sample": 0,
+        #     "type": "image",                   | "annotation"
+        #     "base_resolution": 0,
+        #     "datatype": "uint16",
+        #     "creator": "vikramc",
+        #     "sources": [],
+        #     "downsample_status": "DOWNSAMPLED",
+        #     "related": []
+        # }
+        channel_url = "/".join(
+            [
+                domain,
+                "v1",
+                "collection",
+                collection,
+                "experiment",
+                experiment,
+                "channel",
+                channel,
+            ]
+        )
+        async with await self.auth_get(channel_url, token_key) as r:
+            return await r.json()
+
+    async def get_downsample(
+        self,
+        domain: str,
+        collection: str,
+        experiment: str,
+        channel: str,
+        token_key: Key,
+    ) -> JSON:
+        # {
+        #     "num_hierarchy_levels": 1,
+        #     "cuboid_size": { "0": [512, 512, 16] },
+        #     "extent": { "0": [264, 160, 228] },
+        #     "status": "DOWNSAMPLED",
+        #     "voxel_size": { "0": [50.0, 50.0, 50.0] }
+        # }
+        downsample_url = "/".join(
+            [domain, "v1", "downsample", collection, experiment, channel]
+        )
+        async with await self.auth_get(downsample_url, token_key) as r:
+            return await r.json()
+
+    async def get_cutout(
+        self,
+        domain: str,
+        collection: str,
+        experiment: str,
+        channel: str,
+        resolution: int,
+        range_box: Box3D,
+        token_key: Key,
+    ) -> bytes:
+
+        url = "/".join(
+            [
+                domain,
+                "v1",
+                "cutout",
+                collection,
+                experiment,
+                channel,
+                str(resolution),
+                *(f"{l}:{r}" for l, r in zip(*range_box)),
+            ]
+        )
+        async with await self.auth_get(url, token_key) as r:
```

Comment:
There are multiple dataformats supported:
https://docs.theboss.io/docs/cutout-get#section-getting-data-volumes

I also tried `application/blosc-python`, which is simpler to implement, but uses `pickle` internally, which makes it slower. `image/jpeg` might be an option to save bandwidth, but I think raw data is fine for now. Opinions?